linkedevents/settings.py

```diff
@@ -600,3 +601,6 @@ def dummy_haystack_connection_for_lang(language_code):
 # Enkora course importer
 ENKORA_API_USER = env("ENKORA_API_USER")
 ENKORA_API_PASSWORD = env("ENKORA_API_PASSWORD")
+
+
+NOREPLY_ADDRESS = env("NOREPLY_ADDRESS")
```

Comment:
Django already has `DEFAULT_FROM_EMAIL`. Using that might be a good alternative? https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DEFAULT_FROM_EMAIL

Reply:
Yes, because it's not empty by default it should be ensured that it's set for each environment with the correct value while a variable like `NOREPLY_ADDRESS` can be used only when necessary. And do we want in all cases to have the default "from" email to be the "noreply" address as would happen if the Django variable would be used?

Personally I would lean towards sticking with the `NOREPLY_ADDRESS` variable, but perhaps we need a final word here. What does @tuomas777  think of this?

Reply:
Discussed this with @harriris-vincit and we decided to go with Django's `DEFAULT_FROM_EMAIL` mostly because it is used for this purpose in several other Helsinki's Django projects already.